python/examples/minimal_build/README.md

```diff
@@ -24,38 +24,42 @@ an eye toward maximum portability across different Linux distributions. This
 may help for contributors debugging build issues caused by their local
 environments.
 
-## Fedora 31
+## Ubuntu 18.04
 
 Build image:
 
 ```
-docker build -t arrow_fedora_minimal -f Dockerfile.fedora
+docker build -t arrow_ubuntu_minimal -f Dockerfile.ubuntu
```

Comment:
From my testing, you need to add a dot (`.`) at the end. This is the "context" argument, which is mandatory at least here.

Reply:
This I cannot tell as I am running podman and I do not need .